config/webpack.dev.config.js

```diff
@@ -85,6 +86,7 @@ module.exports = merge(commonConfig, {
                 plugins: [
                   PostCssAutoprefixerPlugin(),
                   PostCssRTLCSS(),
+                  CustomMediaCSS(),
```

Comment:
We'll need to also make similar changes to the other Webpack configs as well, i.e.:
* `webpack.dev-stage.config.js`
* `webpack.prod.config.js`